test/e2e/network_test.go

```diff
@@ -554,12 +577,14 @@ func (tc *testContext) waitUntilJobSucceeds(name string) error {
 		}
 		if job.Status.Failed > 0 {
 			events, _ := tc.getPodEvents(name)
-			return errors.Errorf("job %v failed: %v", job, events)
+			logs, _ := tc.getLogs(job)
+			return errors.Errorf("job %v failed: %v. Logs for job %v: %s", job, events, job, logs)
```

Comment:
No point in having the `job` value in the string twice.
```suggestion
			return errors.Errorf("job %v failed: %v. Logs: %s", job, events, logs)
```